src/engine/BMInterface.php

```diff
@@ -338,6 +344,111 @@ public function count_pending_games($playerId) {
         }
     }
 
+    /**
+     * Load a brief summary of a game, including players, buttons, score, and status
+     */
+    protected function load_game_summary($gameId) {
+        try {
+            $this->set_message('Game summary retrieved successfully.');
+
+            $query = <<<'NOWDOC'
+                SELECT v.game_id,
+                       g.status_id,
+                       s.name as status,
+                       v.n_target_wins,
+                       v.position,
+                       v.player_id,
+                       v.player_name,
+                       v.button_id,
+                       v.button_name,
+                       v.n_rounds_won,
+                       v.n_rounds_drawn,
+                       v.n_rounds_won >= v.n_target_wins AS is_winner
+                FROM game_player_view AS v
+                LEFT JOIN game AS g
```

Comment:
I've been thinking about these LEFT JOINs, and it might actually be more appropriate to simply add these columns to game_player_view, since they're pretty fundamental bits of information about the game.

If you like the idea, I'll drop the view and recreate it.

Reply:
Yup, that's correct. I'll add those fields to game_player_view then.